projects/ccf-eui/src/app/app.component.html

```diff
@@ -27,7 +27,16 @@
       <ccf-drawer-toggle-button></ccf-drawer-toggle-button>
     </ccf-drawer>
     <ccf-drawer-content>
-      <!-- <ccf-organ-selector [multiselect]="true" [organList]="organList" [selectedOrgans]="[organList[1], organList[2], organList[3]]"></ccf-organ-selector> -->
+      <div [class.closed]="!open" class="selector-drawer">
+        <ccf-organ-selector [multiselect]="true" [organList]="organList" [selectedOrgans]="selectedOrgans"></ccf-organ-selector>
```

Comment:
When selected organs change in ccf-organ-selector, save the new selected organs back out to `selectedOrgans` field in app.componenent.ts